packages/showcase/src/Demo.js

```diff
@@ -85,32 +86,49 @@ const DemoToolbar = ({codeMode, onCodeModeSelect, showTSButton, showJSButton}) =
     }
   };
   return (
-    <ToggleButtonGroup
-      sx={{ 
-        margin: '8px 0'
-      }}
-      exclusive
-      value={codeMode}
-      onChange={onCodeModeSelect}
-    >
-      {showJSButton && <ToggleButton
-        sx={toggleButtonStyles}
-        value={'JS'}
-      >
-        <JavaScriptIcon sx={{ fontSize: 20 }} />
-      </ToggleButton>}
-      {showTSButton && <ToggleButton
-        sx={toggleButtonStyles}
-        value={'TS'}
+    <Box sx={{ display: 'flex', alignItems: 'stretch', justifyContent: 'space-between' }}>
+      <ToggleButtonGroup
+        sx={{ 
+          margin: '8px 0'
+        }}
+        exclusive
+        value={codeMode}
+        onChange={onCodeModeSelect}
       >
-        <TypeScriptIcon sx={{ fontSize: 20 }} />
-      </ToggleButton>}
-    </ToggleButtonGroup>
+        {showJSButton && <ToggleButton
+          sx={toggleButtonStyles}
+          value={'JS'}
+        >
+          <JavaScriptIcon sx={{ fontSize: 20 }} />
+        </ToggleButton>}
+        {showTSButton && <ToggleButton
+          sx={toggleButtonStyles}
+          value={'TS'}
+        >
+          <TypeScriptIcon sx={{ fontSize: 20 }} />
+        </ToggleButton>}
+      </ToggleButtonGroup>
+      {showCollapseButton && (
+        <Button
+          sx={{
+            marginTop: 1,
+            marginBottom: 1
+          }}
+          variant='outlined'
+          onClick={() => {
+            onCollapse();
+          }}
+        >
+          {codeOpen ? 'Hide Code' : 'Show Code'}
+        </Button>
+      )}
+    </Box>
   );
 }
 
-export const Demo = ({ component, rawContent }) => {
+export const Demo = ({ component, rawContent, collapsible }) => {
```

Comment:
I think they should all be collapsible (or at least it should be the default) but there should be a prop which determines whether the initial state is open or closed (probably default to open)